packages/core/nuxt-theme-module/theme/components/Notification.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <transition-group tag="div" class="notifications" :name="isMobile ? 'slide' : 'sf-fade'">
+    <SfNotification
+      v-for="notification in notifications"
+      :key="notification.id"
+      :visible="true"
```

Comment:
You could just use `visible`
`:visible="true"` === `visible`

If so, lets move it to the bottom of attributes